### PR TITLE
Recommend realistic light attenuation

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Recommend a realistic Light.attenuation by displaying a warning in the contrary
- [x] Fix worlds and protos
    - [x] `Altino.proto`: fix headlights attenuation
        - [x] `altino.wbt`: check that it has few influence on the world, even when lights are off and leds are on.
    - [x] `inverted_pendulum.wbt`: fix lights (remove the directional light to see the point light on the robot, drop one of the 2 point lights which is useless, increase the plane dimension, fix the algorithm to have a more stable behavior, follow the correct solid)
    - [x] `Portal.proto`: fix attenuation
        - [x] `portal.wbt`: change the world light to see the LED effect if it is enabled
    - [x] `home.wbt`: simply adapt the attenuation: the result is good looking.
    - [x] Dropped the `pointLightAmbientIntensity`, `pointLightAttenuation` fields from the `FloorLight`, `StreetLight` and`CeilingLight` PROTOs, and check/fix exhaustively the consequences on the worlds containing these PROTOs (at least adjust the intensity):
        - [x] `projects/robots/adept/pioneer3/worlds/pioneer3dx_collision_avoidance.wbt`
        - [x] `projects/robots/adept/pioneer3/worlds/pioneer3dx_with_kinect.wbt`
        - [x] `projects/robots/epfl/lis/worlds/blimp.wbt`: Move boards floating in the air, add more shadows, fix initial viewpoint
        - [x] `projects/robots/irobot/create/worlds/create.wbt`
        - [x] `projects/robots/softbank/nao/worlds/challenge.wbt`
        - [x] `projects/robots/softbank/nao/worlds/nao_indoors.wbt`
        - [x] `projects/samples/contests/rockin/worlds/home.wbt`
        - [x] `projects/samples/environments/indoor/worlds/apartment.wbt`
        - [x] `projects/samples/environments/indoor/worlds/kitchen.wbt`
        - [x] `projects/samples/robotbenchmark/humanoid_marathon/worlds/humanoid_marathon.wbt`
        - [x] `projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt`
        - [x] `projects/samples/robotbenchmark/obstacle_avoidance/worlds/obstacle_avoidance.wbt`
        - [x] `projects/vehicles/worlds/city_night.wbt`
        - [x] `projects/vehicles/worlds/village.wbt`
        - [x] `projects/vehicles/worlds/village_realistic.wbt`
        - [x] `projects/vehicles/worlds/village_winter.wbt`
    - [x] Fix SpotLight attenuation in vehicles head lights: test them in night
- [x] Log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
We already increased this rule at some point, because not quadratic lights are uncommon, and new Lights are already created with a quadratic parameter: https://github.com/omichel/webots/blob/develop/src/webots/nodes/WbPointLight.cpp#L48

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
:eyes:

File: change_logs/ChangeLog.html [hunk 1]
```diff
@@ -50,6 +50,7 @@ <h2><a href='https://cyberbotics.com/doc/blog/Webots-2018-b-release'>Webots R201
 <li><b>Enhancements</b></li>
 <ul>
 <li><font color="red">Improved the Sphere texture mapping that now applies UV mapping.</font></li>
+<li><font color="red">Encourage the use of quadratic attenuation for PointLight and SpotLight nodes by displaying a warning in case the 'attenuation' field does not provide realistic lighting.</font></li>
```
Comment (omichel) [inline change_logs/ChangeLog.html]:
```suggestion
<li><font color="red">Encourage the use of realistic quadratic attenuation for PointLight and SpotLight nodes by displaying a warning in case the 'attenuation' field is not quadratic.</font></li>
```

File: projects/robots/epfl/lis/worlds/blimp.wbt [hunk 2]
```diff
@@ -566,27 +564,6 @@ DEF EPFL_LOGO_2 Solid {
...
-        metalness 0
-      }
-      geometry Box {
-        size 3 1.5 0.01
-      }
-    }
-  ]
-  name "lis logo 1"
-}
 DEF LIS_LOGO_2 Solid {
```
Comment (omichel) [inline projects/robots/epfl/lis/worlds/blimp.wbt]:
```suggestion
DEF LIS_LOGO Solid {
```

Also, the name below could be change to "lis logo" instead of "lis logo 2"

File: projects/samples/howto/controllers/inverted_pendulum/inverted_pendulum.c [hunk 3]
```diff
@@ -28,7 +28,7 @@
 #define MAX_FORCE 40
 #define CENTERING_ANGLE 0.7
 #define CENTERING_FORCE 4
-#define TIME_STEP 64
+#define TIME_STEP 32
```
Comment (omichel) [inline projects/samples/howto/controllers/inverted_pendulum/inverted_pendulum.c]: That seems to be unrelated... But maybe it's fine like that...

File: projects/samples/howto/worlds/inverted_pendulum.wbt [hunk 4]
```diff
@@ -17,11 +17,12 @@ WorldInfo {
 Viewpoint {
   orientation 0.05317585096836776 0.9887920240003923 0.1395086454202659 3.609723118221519
   position -6.644331534525101 4.261264054273784 -9.7759672742021
+  follow "robot:solid"
 }
 TexturedBackground {
 }
 Floor {
-  size 2 60
+  size 2 10000
```
Comment (omichel) [inline projects/samples/howto/worlds/inverted_pendulum.wbt]: Really that big?

File: projects/samples/howto/controllers/inverted_pendulum/inverted_pendulum.c (hunk 3 above)
Comment (fabienrohrer) [reply projects/samples/howto/controllers/inverted_pendulum/inverted_pendulum.c]: Yes, a bit unrelated, but I changed this to improve the behavior of this simulation.

File: projects/samples/howto/worlds/inverted_pendulum.wbt (hunk 4 above)
Comment (fabienrohrer) [reply projects/samples/howto/worlds/inverted_pendulum.wbt]: Yes, when the solid speed increase, it can easily overtakes 1000 meters (due to the dimension of the robot). I would put an infinite plane here if I could.

**Comment by fabienrohrer** [comment]
👍
